modules/kummaBidAdapter.js

```diff
@@ -76,7 +76,7 @@ function impression(slot) {
   };
 }
 function banner(slot) {
-  const size = slot.params.size.toUpperCase().split('X');
+  const size = parseSizesInput(slot.sizes)[0].split('x') || slot.params.size.toUpperCase().split('X');
```

Comment:
this would never be falesy so the 2nd value would never evaluate. 

If that's ok then drop it. otherwise you should handle the other cases properly.